.github/workflows/pack-debian.yml

```diff
@@ -92,3 +92,37 @@ jobs:
           khiops_coclustering -v
       - name: Test Khiops and Khiops Coclustering
         uses: ./.github/actions/test-khiops
+  test-kni-deb:
+    needs: build-deb-package
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu22.04, ubuntu20.04]
+    container:
+      image: ghcr.io/khiopsml/khiops/khiopsdev-${{ matrix.os }}:latest
+    steps:
+      - name: Set environment variables
```

Comment:
I would rename to "Put OS info on the environment"

Reply:
done